types/node/v8/base.d.ts

```diff
@@ -5593,6 +5593,8 @@ declare module "stream" {
             allowHalfOpen?: boolean;
             readableObjectMode?: boolean;
             writableObjectMode?: boolean;
+            readableHighWaterMark?: number;
```

Comment:
According https://nodejs.org/api/stream.html#stream_readable_readablehighwatermark this parameter was added at Node.js V9. Please remove.

Reply:
I'll update this pull to revert this for v8, but thought it was added in 8.4 based on https://github.com/nodejs/node/pull/14636 and  https://github.com/nodejs/node/pull/14811.

Reply:
Oh! Again we have a Node.js problem `documentation VS commits history`.
The right approach for resolving such problem is deep research with opening issue for Node.js Documentation. But in this particular case, it will be time-wasting.
Node.js EOL v8 is coming in three weeks. Because of that, my proposal is not touching @types/node@8.

Reply:
it's documented in the v8 version of node docs: https://nodejs.org/dist/latest-v8.x/docs/api/stream.html#stream_readable_readablehighwatermark
If an APi change is backportet the docs on master are usually not updated to include the actual version where it was backported as it would require one more commit on master triggered by a release on another branch.